qa/SidechainTestFramework/sidechainauthproxy.py

```diff
@@ -58,6 +66,11 @@ def __init__(self, service_url, service_name=None, timeout=HTTP_TIMEOUT, connect
         authpair = user + b':' + passwd
         self.__auth_header = b'Basic ' + base64.b64encode(authpair)
 
+        (self._user, self._passwd) = (self.__url.username, self.__url.password)
```

Comment:
I see in scutil.py the following definition of url: ` url = "http://rt:rt@%s:%d" % ('127.0.0.1' or rpchost, sc_rpc_port(i))`
does it have sense to have default credentials `rt:rt` ?

Reply:
Done